imsv-docs-docusaurus/openapi/endpoints/prerequisites/spending-prerequisites.yaml

```diff
@@ -132,4 +133,4 @@ post:
       content:
         application/json:
           schema:
-            $ref: '../../models/api-error-400.yaml'
```

Comment:
Yaml strings use double quotes.

Reply:
this PR switches this line to double quotes